src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -919,6 +923,15 @@ function hasWarningMessage() {
     return $(WARNING_STATUS_MESSAGE).length;
 }
 
+function isSessionClosingSoon() {
+    const endDate = new Date($(END_TIME).html().trim());
+    const currentDate = new Date();
+    if (endDate - currentDate <= MS_IN_FIFTEEN_MINUTES && endDate - currentDate > 0) {
```

Comment:
`endDate - currentDate` can be extracted into a variable `remainingTime` since it's used twice.